sound_play/test/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-add_executable(test_sound_play test.cpp)
-target_link_libraries(test_sound_play ${catkin_LIBRARIES})
-add_dependencies(test_sound_play sound_play_gencpp)
-set_target_properties(test_sound_play PROPERTIES OUTPUT_NAME test)
+# add_executable(test_sound_play test.cpp)
+# target_link_libraries(test_sound_play ${catkin_LIBRARIES})
+# add_dependencies(test_sound_play sound_play_gencpp)
+# set_target_properties(test_sound_play PROPERTIES OUTPUT_NAME test)
```

Comment:
add them directly in the follow up PR

Reply:
Just to clarify, do you mean to delete all this test related stuff on this PR so we can tackle them appropriately on the follow up PR?

Reply:
Yeaah